AppLibraryEnabler.xm

```diff
@@ -127,26 +139,40 @@
 	%orig;
 	UIView *containerView = [self containerView];
 	CGRect containerFrame = containerView.frame;
-	[self.view setFrame:containerFrame];
+	self.view.frame = containerFrame;
 }
 %end
 
-//	%hook _SBHLibraryPodIconListView
-//	- (CGRect)frame {
-//		CGRect origValue = %orig;
-//		CGRect newContainerFrame = origValue;
-//		newContainerFrame.size.width = ?;
-//		return newContainerFrame;
-//	}
-//	%end
-
-extern "C" bool _os_feature_enabled_impl(const char *domain, const char *feature);
-%hookf(bool, _os_feature_enabled_impl, const char *domain, const char *feature) {
-	if (strcmp(domain, "SpringBoard") == 0 && strcmp(feature, "Dewey") == 0) {
-		return true;
-	}
-	return %orig;
-}
```

Comment:
I find that this feature flag is enabled on iPad, thus no longer needed. You may want to verify if this is the case for older iPad models.